src/app/auth/login/login.component.html

```diff
@@ -1,8 +1,8 @@
 <div class="login-container bg-light">
   <div class="login-box">
-    <h1 translate>APP_NAME</h1>
+    <h1>Log in to App</h1>
     <div>
-      <h6 class="d-inline-block">v{{ version }}</h6>
+      <h6 class="d-inline-block"></h6>
```

Comment:
```suggestion
      <h6 class="d-inline-block">v{{ version }}</h6>
```
this one is useful for checking that the correct version is deployed